vuepress/docs/next/tutorials/create/pb/publish-project-bundle.md

```diff
@@ -155,13 +155,18 @@ ent prj install
 ent prj install --conflict-strategy=OVERRIDE
 ```
 
+  > **Note**
+  >
+  > In the event of a timeout or pod failure, refer to the errors captured by the cm and plugin deployer logs. 
```

Comment:
Might need to expand cm. In a quickstart env the pod is something like "quickstart-cm-deployment-ID", so perhaps cm -> quickstart-cm-deployment, to make it clearer. The plugin deployer has a generated name so maybe we leave that one alone.